index.js

```diff
@@ -0,0 +1,13 @@
+function calculateTotal(items) {
+  items.forEach(item => Calculate(item))
```

Comment:
I'd use `map` but `forEach` works too I guess

Reply:
I noted that down and used map today for head-of-the-class.